apps/dashboard/src/lib/features/course/api/newsfeed.svelte.ts

```
@@ -140,14 +140,16 @@ export class NewsfeedApi extends BaseApiWithErrors {
 
           const newFeed = {
             ...response.data,
+            createdAt: response.data.createdAt ?? new Date().toISOString(),
```

ESLint

[error] Found a mutable instance of the built-in Date class. Use SvelteDate instead.

(svelte/prefer-svelte-reactivity)